src/fidesctl/cli/utils.py

```diff
@@ -40,10 +40,13 @@ def check_server(cli_version: str, server_url: str) -> None:
         raise SystemExit(1)
 
     server_version = health_response.json()["version"]
-    if str(server_version) == str(cli_version):
-        echo_green(
-            "Server is reachable and the client/server application versions match."
-        )
+    if str(server_version) == str(cli_version) or f"{server_version}.dirty" == str(
+        cli_version
+    ):
```

Comment:
```suggestion
    if str(server_version).removesuffix(".dirty") == str(cli_version):
```

Reply:
actually I just realized `removesuffix` was introduced in python 3.9, and our dockerfile uses 3.8?

Reply:
I saw https://github.com/ethyca/fides/issues/495 was closed, but it looks like it wasn't actually resolved 😡

Reply:
closed != implemented

we'll be sticking to python 3.8 for the time being